.dockerignore

```diff
@@ -1,6 +1,13 @@
-target
-Dockerfile
-.dockerignore
-.git
-.github
-.gitignore
+# Exclude everything:
+#
+*
+
+# Now un-exclude required files and folders:
+#
+!.cargo
+!*.toml
+!*.lock
+!tower-*
+!zebra-*
+!zebrad
+!docker/entrypoint.sh
```

Comment:
We'll have to remember to update this inclusion list when/if we add new stuff (perhaps)